Command/UpdateClientCommand.php

```diff
@@ -92,6 +99,10 @@ private function updateClientFromInput(Client $client, InputInterface $input): C
     {
         $client->setActive(!$input->getOption('deactivated'));
 
+        if ($input->hasOption('confidential')) {
```

Comment:
@florianajir Keep in mind that `$input->hasOption` check whether the option has been added to the definition (with `addOption`) and **not** if it has been passed during execution (`--confidential`), so i don't think you're getting the expected behavior here.